telegram/src/main/kotlin/com/github/kotlintelegrambot/entities/inputmedia/InputMedia.kt

```diff
@@ -76,8 +76,8 @@ data class InputMediaAnimation(
     @SerializedName(InputMediaFields.HEIGHT) val height: Int? = null,
     @SerializedName(InputMediaFields.DURATION) val duration: Int? = null
 ) : InputMedia() {
-    override val type: String
-        get() = InputMediaTypes.ANIMATION
+    @SerializedName(InputMediaFields.TYPE)
```

Comment:
Nice one! I imagine we can now remove the code handling this manually in `InputMediaAdapter`?https://github.com/kotlin-telegram-bot/kotlin-telegram-bot/blob/main/telegram/src/main/kotlin/com/github/kotlintelegrambot/network/serialization/adapter/InputMediaAdapter.kt#L24-L25

Reply:
I think the entire custom serializer ```InputMediaAdapter``` for the ```InputMedia``` types can be removed.